File: src/main/java/vocadb/notification/reader/client/query/LanguagePreference.java
package vocadb.notification.reader.client.query;

import com.fasterxml.jackson.annotation.JsonProperty;

public enum LanguagePreference {
    @JsonProperty("Default")
    DEFAULT,
    @JsonProperty("Japanese")
    JAPANESE,
    @JsonProperty("Romaji")
    ROMAJI,
    @JsonProperty("English")
    ENGLISH;
}

File: src/main/java/vocadb/notification/reader/client/api/enums/LanguagePreference.kt
package vocadb.notification.reader.client.api.enums

import com.fasterxml.jackson.annotation.JsonValue

enum class LanguagePreference(@JsonValue val value: String) {
    DEFAULT("Default"),
    JAPANESE("Japanese"),
    ROMAJI("Romaji"),
    ENGLISH("English");
}
